app/views/layouts/_main.html.erb

```diff
@@ -17,7 +17,7 @@
 
   <%= login_form.header_html %>
 </head>
-<body>
+<body class="<%= uppercase_enabled?(current_user, Organization.current) ? 'mu-uppercase' : '' %>">
```

Comment:
You can is plain suffix if instead of ternary if: ` 'mu-uppercase' if uppercase_enabled?(....) `. `nil` is translated to blank.

Reply:
Good! We actually have a few of those.

https://github.com/mumuki/mumuki-laboratory/blob/1bd388527ee18bc71b35157abbf003f84225720c/app/views/layouts/_main.html.erb#L27